bin/jobwrapper.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
Please change this to `/usr/bin/env bash` like we use in other files now.  BTW, where is jobwrapper.sh called from?

Reply:
`jobwrapper.sh` called from ebuild modified PATH. When the build system try to call gcc / clang, they actually called this fake script to handle the corresponding command spawn.